force-app/main/default/layouts/SMS__c-HOT SMS Layout.layout-meta.xml

```diff
@@ -19,6 +19,10 @@
                 <behavior>Edit</behavior>
                 <field>Type__c</field>
             </layoutItems>
+            <layoutItems>
+                <behavior>Readonly</behavior>
```

Comment:
Bør vel være Edit-her, og heller begrense mulighet for å redigerte via tillatelsessettet.